test/base/MockNavigator.scala

```diff
@@ -21,21 +21,26 @@ import models.requests.JourneyRequest
 import org.mockito.ArgumentMatchers._
 import org.mockito.BDDMockito._
 import org.mockito.{ArgumentCaptor, Mockito}
-import org.scalatestplus.mockito.MockitoSugar
 import org.scalatest.{BeforeAndAfterEach, Suite}
+import org.scalatestplus.mockito.MockitoSugar
 import play.api.mvc.{AnyContent, Call, Result}
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.http.logging.Authorization
 
 trait MockNavigator extends MockitoSugar with BeforeAndAfterEach { self: MockitoSugar with Suite =>
 
   protected val navigator: Navigator = mock[Navigator]
   protected val aRedirectToTheNextPage: Result = mock[Result]
+  protected val hc: HeaderCarrier = HeaderCarrier(
+    authorization = Some(Authorization(TestHelper.createRandomString(255)))
```

Comment:
Pretty sure you can just do HeaderCarrier() to create a new one

Reply:
I checked it previously and unfortunately `authorization` is necessary, because in other cases you won't be authorised. For us, it's test so it will be easier to use proposal from @ed0906 . Can we add this `Some(Authorization(TestHelper.createRandomString(255)))` as default value if we not provide anything?

Second question, I remember that there was some problems with `createRandomString` method, I'm not sure that exactly with this, but we had Chinese characters or something similar, should we use here method from `TestHelper.createRandomAlphanumericString`?

Reply:
This has been addressed.